### PR TITLE
When --net=none enabled, the test uses local filesystem

### PR DESCRIPTION
**Pull request by brahmaroutu**
Addresses #10963
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
awesome!!

File: integration-cli/docker_cli_build_test.go [hunk 1]
```diff
@@ -770,16 +770,30 @@ RUN [ $(ls -l /exists/exists_file | awk '{print $3":"$4}') = 'dockerio:dockerio'
 }
 
 func TestBuildCopyAddMultipleFiles(t *testing.T) {
-	name := "testcopymultiplefilestofile"
-	defer deleteImages(name)
-	ctx, err := fakeContext(`FROM busybox
+        var url string
+        if os.Getenv("DOCKER_NET") == "none" {
```
Comment (ahmetalpbalkan) [inline integration-cli/docker_cli_build_test.go]: AFAIK we will try not to use environment variables set in the Dockerfile. #10797 check with @tianon

Comment (jfrazelle) [inline integration-cli/docker_cli_build_test.go]: we dont need the if, just run the storage server everytime

**Comment by brahmaroutu** [comment]
@jfrazelle I took out the if statement.

File: integration-cli/docker_cli_build_test.go [hunk 2]
```diff
@@ -770,16 +770,25 @@ RUN [ $(ls -l /exists/exists_file | awk '{print $3":"$4}') = 'dockerio:dockerio'
 }
 
 func TestBuildCopyAddMultipleFiles(t *testing.T) {
+	server, err := fakeStorage(map[string]string{
+		"robots.txt": "hello",
+		"index.html": "world",
```
Comment (ahmetalpbalkan) [inline integration-cli/docker_cli_build_test.go]: `index.html` is not used.

**Comment by brahmaroutu** [comment]
@ahmetalpbalkan I removed the extra index.html Please review.

**Comment by ahmetalpbalkan** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM thanks @brahmaroutu

**Pull request merged**